ntbs-service/Pages/Alerts/TransferRequestAction.cshtml

```diff
@@ -18,17 +18,23 @@
 
         <br>
 
-        <div class="transfer-request-information"> Sending TB service:  @Model.TransferAlert.TbServiceName</div>
-        
-        <div class="transfer-request-information"> Sending case manager:  @Model.Notification?.Episode.CaseManagerName </div>
+        <div class="transfer-request-information flex-container"> 
+            <div class="transfer-request-information-label"> Sending TB service: </div>
+            <div> @Model.TransferAlert.TbServiceName </div>
+        </div>
+
+        <div class="transfer-request-information flex-container"> 
+            <div class="transfer-request-information-label"> Sending case manager: </div>
+            <div> @Model.Notification?.Episode.CaseManagerName </div>
+        </div>
         
-        <div class="flex-container transfer-request-information">
-            <div class="action-transfer-request-label bold-text"> Reason for transfer: </div>
+        <div class="transfer-request-information flex-container">
+            <div class="transfer-request-information-label--no-wrap bold-text"> Reason for transfer: </div>

```

Comment:
`--no-wrap` tends to be a modifier to add to or overrule the non `--` part of the class. Here it's just setting distinct values - if I could think of a better name for it I would suggest it. Seeing as I can't feel free to ignore this comment.